sdk/communication/Azure.ResourceManager.Communication/src/Generated/CommunicationServiceResourceCollection.cs

```diff
@@ -37,8 +39,14 @@ protected CommunicationServiceResourceCollection()
         /// <summary> Initializes a new instance of the <see cref="CommunicationServiceResourceCollection"/> class. </summary>
         /// <param name="client"> The client parameters to use in these operations. </param>
         /// <param name="id"> The identifier of the parent resource that is the target of operations. </param>
-        internal CommunicationServiceResourceCollection(ArmClient client, ResourceIdentifier id) : base(client, id)
```

Comment:
I think this is unexpected changes due to the type of `subscriptionid` was changed from `string` to `Guid`.